front-end/src/Footer.css

```diff
@@ -1,17 +1,59 @@
 @media screen and (max-width: 2400px) {
-  .footer-links {
+  .footer-links,
+  #trakt-power {
     font-size: 18px;
   }
+
+  #trakt-logo {
+    width: 3%;
+  }
+}
+
+@media screen and (max-width: 1600px) {
+  .footer-links,
+  #trakt-power {
+    font-size: 17px;
+  }
+
+  #trakt-logo {
+    width: 4%;
+  }
 }
 
 @media screen and (max-width: 992px) {
-  .footer-links {
+  .footer-links,
+  #trakt-power {
     font-size: 16px;
   }
+
+  #trakt-logo {
+    width: 5%;
+  }
 }
 
 @media screen and (max-width: 650px) {
-  .footer-links {
+  .footer-links,
+  #trakt-power {
     font-size: 14px;
```

Comment:
for small width, the 'powered by trakt' overlaps with the 'meet the team'; I can try to edit a bit later or you can also try to debug using developer tools in browser hehe, sorry

Reply:
I'll take a look at it

Reply:
should be good now